### PR TITLE
Added taxon GCI to between organogenesis and neurula stage.

### LINKED ISSUE #2828: Many stage relations should be taxon GCIs
I am using graph-based taxon constraints in OAK (https://github.com/INCATools/ontology-access-kit/blob/main/notebooks/Commands/TaxonConstraints.ipynb) and I am seeing lots of GO terms restricted to vertebrates

e.g. here is a path:

` cell proliferation involved in compound eye morphogenesis|part of|compound eye morphogenesis|part of|compound eye development|results in development of|compound eye|None|multi-unit eye|None|eye|part of|visual system|capable of part of|visual perception|None|sensory perception of light stimulus|None|sensory perception|None|nervous system process|has participant|nervous system|immediate transformation of|future nervous system|existence starts during or after|organogenesis stage|preceded by|neurula stage|simultaneous with|neural tube formation|part of|neural tube development|results in development of|neural tube|develops from|neural rod|develops from|neural keel|develops from|neural plate|developmentally induced by|notochord|part of|axial skeletal system|has part|axial skeleton plus cranial skeleton|develops from|sclerotome`

organogenesis seems pan metazoan but neurulas are vertebrate

See also

- https://github.com/obophenotype/developmental-stage-ontologies/issues/84

### PR DESCRIPTION
**Pull request by cmungall**
Fixes #2828

Fixes https://github.com/obophenotype/developmental-stage-ontologies/issues/84

### CODE REVIEW

**Comment by anitacaron** [comment]
Hi @cmungall, could you please request someone to review this PR? I'll run a new one tomorrow if you want it to be in the next release.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -78648,8 +78652,8 @@ synonym: "field CA1 of hippocampus" RELATED [NeuroNames:183]
 synonym: "field CA1, Ammon's horn (Lorente de Ns)" RELATED [NeuroNames:183]
 synonym: "hippocampus CA1" EXACT []
 synonym: "prosubiculum = distal ca1" EXACT [BIRNLEX:1197]
-synonym: "regio i cornus ammonis" EXACT LATIN [FMA:74042, FMA:TA]
-synonym: "regio i hippocampi proprii" EXACT LATIN [FMA:74042, FMA:TA]
+synonym: "regio I cornus ammonis" EXACT LATIN [FMA:74042, FMA:TA]
+synonym: "regio I hippocampi proprii" EXACT LATIN [FMA:74042, FMA:TA]
```
Comment (balhoff) [inline src/ontology/uberon-edit.obo:78656]: This change seems unrelated.

**Review by balhoff — APPROVED** [review]
The GCI looks good. There is another change which seems unrelated to this issue—capitalizing an 'i'.

**Comment by cmungall** [comment]
apologies for not separating concerns in the PR!
